src/Arcus.Scripting.DevOps/Scripts/Save-AzDevOpsBuild.ps1

```diff
@@ -1,28 +1,26 @@
 param(
    [Parameter(Mandatory = $true)][string] $ProjectId = $(throw "ProjectId is required"),
-   [Parameter(Mandatory = $true)][string] $BuildId = $(throw "BuildId is required")   
+   [Parameter(Mandatory = $true)][string] $BuildId = $(throw "BuildId is required")
 )
 
-$retentionPayload = @{        
+$retentionPayload = @{
   keepforever='true'
 }
 
 $requestBody = $retentionPayload | ConvertTo-Json -Depth 1 -Compress
 
 $collectionUri = $env:SYSTEM_COLLECTIONURI
-
-if( $collectionUri.EndsWith('/') -eq $False ){
+if ($collectionUri.EndsWith('/') -eq $false) {
```

Comment:
I love the whitespace that has been removed just before this if actually :)
https://www.frankwatching.com/archive/2018/12/04/de-kracht-van-witruimte/

Reply:
F# for the win 😉 . I see your point, same comment as the one before. We can talk about this, but it's important to streamline this. All other Arcus repo's don't do this too.